src/main/java/com/mrcrayfish/controllable/client/settings/ControllerOptions.java

```diff
@@ -23,6 +23,14 @@
 {
     private static final DecimalFormat FORMAT = new DecimalFormat("0.0#");
 
+    public static final SliderPercentageOption ICONS = new ControllableSliderPercentageOption("controllable.options.icons", 0, EnumIcons.values().length - 1, 1, gameSettings -> {
```

Comment:
I've added a new class ControllableEnumOption which is better than using a slider to change an enum.